### PR TITLE
Jv hetero enrich

### PR DESCRIPTION
**Pull request by 1aviervargas**
Working in enRich staff of Moshen

### CODE REVIEW

File: pyworkflow/em/packages/relion/convert.py [hunk 1]
```diff
@@ -553,6 +553,10 @@ def rowToParticle(partRow, **kwargs):
     if partRow.hasLabel(md.RLN_PARTICLE_ID):
         img._rlnParticleId = Integer(partRow.getValue(md.RLN_PARTICLE_ID))
     
+    # copy particleId if available from row to particle
+    if partRow.hasLabel(md.RLN_PARTICLE_RANDOM_SUBSET):
+        img._rln_halfId = Integer(partRow.getValue(md.RLN_PARTICLE_RANDOM_SUBSET))
```
Comment (josuegbl) [inline pyworkflow/em/packages/relion/convert.py]: should have img._rln._halfId. To assign no standards properties, the pattern is something like: obj._package._property

File: pyworkflow/em/packages/xmipp3/protocol_volume_homogenizer.py [hunk 2]
```diff
@@ -29,153 +30,356 @@
...
-                           "Nyquist theorem.\n"
-                           "Note:\n"
-                           "Based on the experimental results, the best value "
-                           "for cut-off frequency is 20A "
-                           "(digitalFrequency = samplingRate/20)")
+        
+        form.addParam('dofrm', params.BooleanParam, default=True,
+                      condition='doAlignment',
+                      label='Use Fast Rotational Matching.',
+                      help="Use Fast Rotational Matching. Before use it you have to install it by scipion install frm"
```
Comment (josuegbl) [inline pyworkflow/em/packages/xmipp3/protocol_volume_homogenizer.py]: Please, correct all code to standard PEP8

File: pyworkflow/em/packages/relion/convert.py (hunk 1 above)
Comment (delarosatrevin) [reply pyworkflow/em/packages/relion/convert.py]: @josuegbl  I think that you have an extra '.' here. Should be something like '_rln_halfId'

**Comment by 1aviervargas** [comment]
@josuegbl @delarosatrevin  ready to merge

**Pull request merged**